Artifacts.toml

```diff
@@ -0,0 +1,42 @@
+[[ncbi-blast]]
+arch = "aarch64"
+git-tree-sha1 = "0406b91031ce302fa9117606d007d04635279fef"
+os = "macos"
+
+    [[ncbi-blast.download]]
+    sha256 = "e8da4f34b0421da051b1c2d9e5f8e1e43bf76e38cf9361a1dbe2fbc44b6224db"
+    url = "https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.16.0/ncbi-blast-2.16.0+-aarch64-macosx.tar.gz"
+[[ncbi-blast]]
+arch = "x86_64"
+git-tree-sha1 = "690abc170201bc029d0e7386da0791d7a76d98ff"
+os = "macos"
```

Comment:
Is this valid TOML? looks like it would overwrite the entries above...

Though I assume this was auto-generated by Artifacts.jl, so maybe I'm misunderstanding

Reply:
Yeah, it is not obvious, but the double brackets is toml syntax for "array of tables" https://toml.io/en/v1.0.0#array-of-tables